src/vstart.sh

```diff
@@ -231,8 +232,9 @@ options:
 	-X disable cephx
 	-g --gssapi enable Kerberos/GSSApi authentication
 	-G disable Kerberos/GSSApi authentication
-	--hitset <pool> <hit_set_type>: enable hitset tracking
-	-e : create an erasure pool
+    --hitset <pool> <hit_set_type>: enable hitset tracking
+    -e : create an erasure pool
+    --exporter: start the ceph-exporter daemon
```

Comment:
As above, suggest `--cephexporter` to be explicit, and line 377 as well.

Reply:
Yeah i have made the changes wherever necessary. Please check now @anthonyeleven